models/migrations/migrations.go

```diff
@@ -1,4 +1,5 @@
 // Copyright 2015 The Gogs Authors. All rights reserved.
+// Copyright 2017 The Gitea Authors. All rights reserved.
```

Comment:
@techknowlogick 2019

Reply:
This one was kept as 2017 because it was changed then, but the copyright was missed in other PRs.